core/persister/src/main/resources/application.conf

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+akka.kafka.consumer {
+  # Properties defined by org.apache.kafka.clients.consumer.ConsumerConfig
+  # can be defined in this configuration section.
+  kafka-clients {
+    # Disable auto-commit by default
+    enable.auto.commit = false
+
+    # What to do when there is no initial offset in Kafka or if the current offset
+    # does not exist any more on the server (e.g. because that data has been deleted):
+    # - earliest : automatically reset the offset to the earliest offset
+    # - latest   : automatically reset the offset to the latest offset
+    # - none     : throw exception to the consumer if no previous offset is found for the consumer's group
+    auto.offset.reset = "earliest"
```

Comment:
This implies the multiple processing of the same activation.
Some activation stores would just override existing activations but others may not.

Reply:
Yes the semantics being implemented here are at-least-once processing. So same activation may be processed multiple times